liquidcore/twofactor/devices.py

```diff
@@ -13,6 +13,15 @@ def create(user):
     )
 
 
+def add(user):
+    return TOTPDevice.objects.create(
+        user=user,
+        name=user.get_username() +
+        str(sum(1 for device in TOTPDevice.objects.devices_for_user(user))),
```

Comment:
We don't want multiple devices for one user. Let's enforce this in this function too, by doing something like `get_or_create` instead of `create` with some name that's exactly the username.